e2e/js/src/js-swc.test.ts

```diff
@@ -134,8 +134,9 @@ export function x() {
 
     // update .swcrc to use path mappings
     updateJson(`libs/${lib}/.swcrc`, (json) => {
+      json.jsc.baseUrl = '.';
       json.jsc.paths = {
-        'src/*': ['src/*'],
+        '~/*': ['./src/*'],
```

Comment:
comment: so the key is to use relative path only for path mappings with swc config file?

Reply:
yes, and the `baseUrl` is only added to pass CI. It's not needed locally (MacOS/Debian), but seems to be needed in CI even with the same swc versions.